docs/request.schema.json

```diff
@@ -25,18 +25,11 @@
                 "/index.html?foo=bar"
             ]
         },
-        "resourcePath": {
-            "type": "string",
-            "description": "The resource path (without extension) that has been requested",
-            "examples": [
-                "/index"
-            ]
-        },
```

Comment:
why do you remove this? @trieloff, should we include it via fastly, compute it, or drop it completely? resource is a _old_ concept from sling, and we don't use it so far. :-)

Reply:
what is the difference between `path` and `resourcePath+selector+extension` ?

Reply:
I'd drop it. We can't do proper (read: Sling-like) resource resolution anyway and it would only be a duplication of the owner, repo, ref, path quadruple that we use to address content.

Reply:
ok